Buildings/Fluid/Storage/Plant/UsersGuide.mo

```diff
@@ -0,0 +1,104 @@
+within Buildings.Fluid.Storage.Plant;
```

Comment:
I think this may have been discussed before.  But perhaps it is better to locate this new package in ``Buildings.DHC.Plants.Cooling`` since it is specifically built for a plant located in a district cooling network with possibility for remote charging?

Reply:
Tagging @mwetter, would you agree?

Reply:
Since this model is specifically for cooling plants, I think users would search for it in `DHC.Plants.Cooling` and hence that would be the preferred package to put it in.

Reply:
Sounds good thanks.  Then @hcasperfu can you move it to  ``Buildings.Experimental.DHC.Plants.Cooling``?

Reply:
I can certainly do this!
Looking at this package,

<img width="165" alt="image" src="https://github.com/lbl-srg/modelica-buildings/assets/91219266/86184639-af8a-4b50-ac28-3bab3fa89d7d">

should I merge the new subpackage so that we have something like
`DHC.Cooling.Plants.{ElectricChillerParallel,StoragePlant}`,
or should I separate them so that we have something like
`DHC.Cooling.Plants.{Electric.ChillerParallel,Storage.PlantWithTank}`?

Reply:
Thank you.  I would do ``DHC.Cooling.Plants.{ElectricChillerParallel,StoragePlant}``.

Reply:
Moved in 71d9673.
Related to this, I also removed some unnecessary subclasses in 1658dd9 and 07d9352 so that the package is less bloat.